src/CSS/styles.css

```diff
@@ -0,0 +1,201 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  background-color: hsl(211.76, 92.73%, 10.78%);
+}
+
+h1 {
+  font-family: "Creepster", cursive;
+  color: hsl(93.8, 61.74%, 77.45%);
+  text-align: center;
+  display: block;
+  margin-top: 10%;
+  font-size: 3rem;
+  padding-bottom: 0%;
+}
+
+p {
+  font-family: "Orbitron", sans-serif;
+  font-size: 1rem;
+  color: antiquewhite;
+  text-align: center;
+  margin-left: 5%;
+  margin-right: 5%;
+}
+
+.intro {
+  display: block;
+  margin: auto;
+  width: 50%;
+  text-align: center;
+}
+
+button {
+  display: block;
+  margin: auto;
+  color: white; /* set text color to white */
```

Comment:
As mentioned elsewhere these comments dont really add anything of value, should be removed